tier2/{{cookiecutter.project_slug}}/COMMUNITY_GUIDELINES.md

```diff
@@ -20,13 +20,13 @@ All community members are expected to adhere to our [Code of Conduct](CODE_OF_CO
 
 Information on contributing to this repository is available in our [Contributing file](CONTRIBUTING.md).
 
-When participating in {Project Name} open source community conversations and spaces, we ask individuals to follow the following guidelines:
+When participating in {{ cookiecutter.project_name }} open source community conversations and spaces, we ask individuals to follow the following guidelines:
 
 - When joining a conversation for the first time, please introduce yourself by providing a brief intro that includes:
   - your related organization (if applicable)
   - your pronouns
   - disclosure of any current or potential financial interest in this work
```

Comment:
I believe we can remove this line about financial interests, this concern is covered by other existing CMS/Federal policies.

Reply:
Should I remove this line in COMMUNITY_GUIDELINES.md across all tiers?

Reply:
Nvm, this is a yes!